package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-cookie-banner",
-  "version": "4.0.0",
+  "version": "4.0.1",
```

Comment:
If this is not a bug fix why incrementing the `PATCH` value?

Reply:
@giuscri, How to be right?

Reply:
I'd not modify `package.json`

Reply:
I updated the version to `4.1.0`. In semver terms, this is not really a bug fix (-> increment the patch number in semver), nor a new feature (increment the minor version number), nor a breaking release (increment the major number). This being said, since this package has been dormant for a while now, I believe minor is safer.

In any case, the release version gets updated contextually to the release, not in the PR doing the code changes - you don't usually know what version a given change will end up being released in, and the npm version/publish process does that for you while releasing. So good call on leaving that untouched in this PR